packages/app-mobile/root.tsx

```diff
@@ -820,8 +823,15 @@ class AppComponent extends React.Component {
 	}
 
 	public componentWillUnmount() {
-		RNAppState.removeEventListener('change', this.onAppStateChange_);
-		Linking.removeEventListener('url', this.handleOpenURL_);
+		if (this.appStateChangeListener_) {
+			this.appStateChangeListener_.remove();
+			this.urlOpenListener = null;
```

Comment:
Should be appStateChangeListener_

Reply:
Thank you for catching this! It should be fixed.